app/controllers/api/v1/pages_controller.rb

```diff
@@ -9,6 +9,10 @@ def index
   def create
     new_page = form.pages.new(page_params)
 
+    if FeatureService.enabled?(:accept_legacy_answer_types)
+      convert_old_answer_types_to_new_format(new_page)
+    end
```

Comment:
Does this work? Looking at the method, it seems to convert `page` rather than `new_page`.